amf-api-contract/shared/src/main/scala/amf/apicontract/client/platform/model/domain/bindings/googlepubsub/GooglePubSubMessageBinding.scala

```diff
@@ -0,0 +1,62 @@
+package amf.apicontract.client.platform.model.domain.bindings.googlepubsub
+
+import amf.apicontract.client.platform.model.domain.bindings.{BindingVersion, MessageBinding}
+import amf.apicontract.client.scala.model.domain.bindings.googlepubsub.{GooglePubSubMessageBinding => InternalGooglePubSubMessageBinding, GooglePubSubSchemaDefinition => InternalGooglePubSubSchemaDefinition}
+import amf.core.client.platform.model.StrField
+import amf.core.client.platform.model.domain.{DomainElement, ObjectNode}
+import amf.apicontract.internal.convert.ApiClientConverters._
+
+import scala.scalajs.js.annotation.{JSExportAll, JSExportTopLevel}
+
+
+@JSExportAll
+case class GooglePubSubMessageBinding(override private[amf] val _internal: InternalGooglePubSubMessageBinding)
+  extends MessageBinding
+  with BindingVersion {
+  @JSExportTopLevel("GooglePubSubMessageBinding")
+  def this() = this(InternalGooglePubSubMessageBinding())
+
+  override protected def bindingVersion: StrField = _internal.bindingVersion
+  def attributes: ObjectNode = _internal.attributes
+  def orderingKey: StrField = _internal.orderingKey
+  def schema: InternalGooglePubSubSchemaDefinition = new GooglePubSubSchemaDefinition(_internal.schema)
+
+  def withAttributes(attributes: ObjectNode): this.type = {
+    _internal.withAttributes(attributes)
+    this
+  }
+  def withOrderingKey(orderingKey: String): this.type = {
+    _internal.withOrderingKey(orderingKey)
+    this
+  }
+  def withSchema(schema: InternalGooglePubSubSchemaDefinition): this.type = {
```

Comment:
Should receive GooglePubSubSchemaDefinition, not InternalGooglePubSubSchemaDefinition

Reply:
done